module1/build.gradle.kts

```diff
@@ -16,18 +20,12 @@ buildscript {
 
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion")
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-        classpath("org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion")
     }
 }
 
 apply {
-    plugin("base")
-    plugin("kotlin")
-    plugin("kotlin-spring")
     plugin("org.springframework.boot")
     plugin("io.spring.dependency-management")
```

Comment:
Unfortunately it's not possible to apply the spring plugins using the `plugins {}` block because their development versions (e.g. `2.0.0.M3`) published to their development repositories do not include the [plugin markers artifacts](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers).